### PR TITLE
Change Ansible description

### PR DESCRIPTION
**Pull request by Andersson007**
Relates to https://github.com/ansible/jinja-docsite/issues/66

As in the issue ^
- `Ansible offers powerful and flexible automation that is simple, decentralized, and easy.`: the words `simple` and `easy` feel a bit like a repetition of the same thing, i'm not a native speaker though. Maybe `simple, decentralized, and free`?
- Also the meaning of `decentralized` is not obvious for me in this context. If you have a controller, you have kinda center:) Should we maybe replace it with something `simple, ???, and free` ?

I suggest improving it a bit

### CODE REVIEW

File: data/pages.yaml [hunk 1]
```diff
@@ -5,7 +5,7 @@ ansible_pkg:
 # Content for the index page
 index:
   title: Ansible community documentation
-  intro: Ansible offers powerful and flexible automation that is simple, decentralized, and easy.
+  intro: Ansible offers powerful and flexible automation that is simple, community-powered, and free.
```
Comment (oraNod) [inline data/pages.yaml:8]:
```suggestion
  intro: Ansible offers open-source automation that is simple, flexible, and powerful.
```

Comment (oraNod) [reply data/pages.yaml:8]: I agree with changing the statement but your proposal uses the word "power" twice in the same sentence.

Comment (Andersson007) [reply data/pages.yaml]: Ah, good point, totally agreed (was also thinking about adding `free`), thanks, your suggestion LGTM. Committed, PTAL

**Comment by oraNod** [comment]
Thanks @Andersson007

**Comment by Andersson007** [comment]
@oraNod thanks for reviewing!

**Pull request merged**